lib/llvmopencl/Workgroup.cc

```diff
@@ -445,6 +445,7 @@ bool WorkgroupImpl::runOnModule(Module &M, llvm::FunctionAnalysisManager &FAM) {
     F.removeFnAttr(Attribute::StrictFP);
     F.removeFnAttr("denormal-fp-math");
     F.removeFnAttr("denormal-fp-math-f32");
+    F.removeFnAttr("no-builtins");
```

Comment:
Pls add a comment why this is removed here so we won't forget.

Reply:
done